themes/src/main/resources/theme/base/admin/resources/partials/realm-tokens.html

```diff
@@ -152,4 +152,4 @@
 
 </div>
 
-<kc-menu></kc-menu>
```

Comment:
@filipelautert, if you remove the changes to this file (```realm-tokens.html```), I can merge this PR right away.